app/assets/javascripts/pb_graph.coffee

```diff
@@ -0,0 +1,42 @@
+$ ->
+  if not gon.run.raw_splits?
+    return
```

Comment:
looks like we need these two lines in all the other files as well, i'm getting console errors from the graph code trying to run on other pages.

i also think we should find a cleaner way of only running this code on the graphs page. if another page starts using a `raw_splits` field for some other reason, then this code will run even though the page doesn't want graphs.